app/models/wpcc/salary_message.rb

```diff
@@ -0,0 +1,28 @@
+module Wpcc
+  class SalaryMessage
+    include ActiveModel::Model
+
+    attr_accessor :salary, :salary_frequency
+
+    TAX_RELIEF_THRESHOLD_RATE = {
+      year: 11_500,
+      month: 958.33,
+      fourweeks: 884.61,
+      week: 221.15
+    }.freeze
+
+    def salary_below_tax_relief_threshold?
+      valid_salary_frequency? && salary_below_frequency_threshold?
+    end
+
+    private
+
+    def valid_salary_frequency?
+      TAX_RELIEF_THRESHOLD_RATE.keys.include?(salary_frequency.to_sym)
```

Comment:
Just a note that we need to be careful calling #to_sym The best approach is to simply never convert user-controlled input to a Symbol, but it is okay on this case because we are using a list of acceptable values. The reason to avoid using to_sym is because Symbols are not garbage collected, creation of large numbers of Symbols could lead to a server running out of memory pretty easily.